controller/workitem.go

```diff
@@ -586,9 +586,12 @@ func ConvertWorkItem(request *http.Request, wit workitem.WorkItemType, wi workit
 			if description != nil {
 				op.Attributes[name] = (*description).Content
 				op.Attributes[workitem.SystemDescriptionMarkup] = (*description).Markup
-				// let's include the rendered description while 'HTML escaping' it to prevent script injection
-				op.Attributes[workitem.SystemDescriptionRendered] =
-					rendering.RenderMarkupToHTML(html.EscapeString((*description).Content), (*description).Markup)
+				renderedDescription := rendering.RenderMarkupToHTML((*description).Content, (*description).Markup)
+				// Escape text only if it is PlainText, Markdown content is already escaped by RenderMarkupToHTML
+				if op.Attributes[workitem.SystemDescriptionMarkup] == rendering.SystemMarkupPlainText {
```

Comment:
what about doing this call to `html.EscapeString` in https://github.com/fabric8-services/fabric8-wit/blob/HEAD/rendering/markup_render.go#L22 instead ? that would make it centralized, IMO.

Reply:
Yes. Thank you, I should've added it there in the first place. 
I've updated it in https://github.com/fabric8-services/fabric8-wit/pull/2165/commits/5f2b92330cbd15457022d86d22a9f47deea76649#diff-45267eee3b1f62d2b019b831a0a0bcfdR23